### PR TITLE
Feat/hng #22 external static landing page

### PR DESCRIPTION
**Pull request by Ayodejioladimeji**
# Description

**Closes #issue_number_here**

# Changes proposed

## What were you told to do?
Feat/hng #22 external static landing page

## What did you do?
Implemented the External Landing page

# Check List (Check all the applicable boxes)

🚨Please review the [contribution guideline](CONTRIBUTING.md) for this repository.

- [X] My code follows the code style of this project.
- [X] This PR does not contain plagiarized content.
- [X] The title and description of the PR is clear and explains the approach.
- [] I am making a pull request against the **dev branch** (left side).
- [X] My commit messages styles matches our requested structure.
- [] My code additions will fail neither code linting checks nor unit test.
- [X] I am only making changes to files I was requested to.

# Screenshots/Videos
Boilerplate Landing Page

### CODE REVIEW

File: src/components/layouts/homepage/styles/hero.css
Comment (prudentbird) [inline src/components/layouts/homepage/styles/hero.css:1]: Use only tailwind, CSS isn't allowed

File: src/components/layouts/homepage/styles/how-it-works.css
Comment (prudentbird) [inline src/components/layouts/homepage/styles/how-it-works.css:1]: Use only tailwind, CSS isn't allowed

**Review by Samadeen — APPROVED** [review]

File: src/components/layouts/homepage/styles/styles.css
Comment (prudentbird) [inline src/components/layouts/homepage/styles/styles.css:1]: Use only tailwind, CSS isn't allowed

File: src/components/layouts/homepage/styles/perfect-fit.css
Comment (prudentbird) [inline src/components/layouts/homepage/styles/perfect-fit.css:1]: Use only tailwind, CSS isn't allowed

File: src/components/layouts/homepage/styles/testimonials.css
Comment (prudentbird) [inline src/components/layouts/homepage/styles/testimonials.css:1]: Use only tailwind, CSS isn't allowed

File: src/components/layouts/homepage/styles/user-section.css
Comment (prudentbird) [inline src/components/layouts/homepage/styles/user-section.css:1]: Use only tailwind, CSS isn't allowed

**Review by prudentbird — CHANGES_REQUESTED** [review]
Resolve styles

**Pull request merged**